account-compression/sdk/src/accounts/index.ts

```diff
@@ -0,0 +1,190 @@
+import { PublicKey, Connection } from "@solana/web3.js";
+import * as borsh from "borsh";
+import * as BN from 'bn.js';
```

Comment:
Potentially yes. But since `beet` deserializes data to `BN` by default, can I skip this for now? 

Most of the TS SDKs I've used have used `BN` instead of native

Reply:
Yeah sure that's fine, just something to keep in mind for the future since BN adds unnecessary bloat.  You may also need to move BN / beet / chai to "dependencies" in package.json since these are not part of tests

Reply:
Great catch, thank you! Moved in latest commit